### PR TITLE
Core docs

### PR DESCRIPTION
**Pull request by swillner**
Fix documentation of the OpenSCM core/low-level API

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -3,35 +3,66 @@ Usage
...
 Time frames
 -----------
 
-Timeseries parameters are always given with a corresponding time frame, which consists of a time point and a period length. The time point gives the start of the timeseries; the period length gives the length of the period between consecutive values in the timeseries. Each parameter value is assumed to be the **average value** for its corresponding period. This implies that additive values such as emissions need to be given as a rate, e.g. ``tC/a`` rather than ``tC``.
+Timeseries parameters are always given with a corresponding time
+frame, which consists of a time point and a period length. The time
+point gives the start of the timeseries; the period length gives the
+length of the period between consecutive values in the timeseries.
+Each parameter value is assumed to be the **average value** for its
+corresponding period. This implies that additive values such as
```
Comment (znicholls) [inline docs/usage.rst]:
```suggestion
corresponding period. This implies that flux variables such as
```

File: docs/usage.rst [hunk 2]
```diff
@@ -58,53 +90,103 @@ A model run is represented by a :class:`openscm.core.Core` object specifying the
...
-
-Using :class:`~openscm.core.ParameterView` as proxy objects rather than directly setting/returning parameter valus allows for efficient parameter handling in the expected units and time frames without specifying these for each value (e.g. seeting a timeseries step-wise would create large overhead).
+In the low-level API parameters are get and set through of subclasses
+of :class:`ParameterView`. While the values of the parameters are
+stored internaly, a :class:`ParameterView` provides an (always
+up-to-date) "view" of the corresponding parameter giving the parameter
+values in a specific unit and, in the case of timeseries, a specific
+time frame.
+
+Unit and time frame have to be specified questing a
```
Comment (znicholls) [inline docs/usage.rst]:
```suggestion
Unit and time frame have to be specified when requesting a
```

File: docs/usage.rst [hunk 3]
```diff
@@ -58,53 +90,103 @@ A model run is represented by a :class:`openscm.core.Core` object specifying the
...
+:class:`~openscm.core.ParameterView` from
```
Comment (znicholls) [inline docs/usage.rst]:
```suggestion
:class:`~openscm.core.ParameterView` from the
```

File: docs/usage.rst [hunk 4]
```diff
@@ -58,53 +90,103 @@ A model run is represented by a :class:`openscm.core.Core` object specifying the
...
+  a writable view to a scalar parameter
+  (:class:`~openscm.core.WritableScalarView`)
+- :func:`~openscm.core.ParameterSet.get_writable_timeseries_view`
+  returns a writable view to a timeseries parameter
+  (:class:`~openscm.core.WritableTimeseriesView`)
+
+Each of these functions take the hierarchical name of the parameter
+(as described under :ref:`parameters`) and, in a similar fashion, the
+hierarchical name of the region it applies to or an empty tuple,
+``()``, in case the parameter applies to all regions.
```
Comment (znicholls) [inline docs/usage.rst]:
```suggestion

```

File: docs/usage.rst (hunk 1 above)
Comment (swillner) [reply docs/usage.rst]: Hmm, sounds non-standard to me. We should find a compromise on how to call these... (this PR is only about reformatting and including additional modules in the docs, though)

File: docs/usage.rst [hunk 5]
```diff
@@ -58,53 +90,103 @@ A model run is represented by a :class:`openscm.core.Core` object specifying the
...
+no standard for the unit and time frame for internal storage, but
+those of the first :class:`ParameterView` requested are used. If a
+scalar view for a time series is requested (or vice-versa), or if the
+units are not convertible, an error is raised.
+
+:class:`~openscm.core.ParameterView` objects also convert between
+hierarchical levels if possible: a view to a higher level parameter
+yields the sum of its child parameters. This implies that, once a
+*writable* view to a parameter is requested, there cannot be a view to
+one of its children. Otherwise consostency cannot be guaranteed, so an
```
Comment (znicholls) [inline docs/usage.rst]:
```suggestion
one of its children. Otherwise consistency cannot be guaranteed, so an
```

File: docs/usage.rst (hunk 1 above)
Comment (znicholls) [reply docs/usage.rst]: maybe flux is too closely related to electromagnetism, how about

```suggestion
corresponding period. This implies that values which define the flow/transport of a quantity (such as emissions, flow of a gas, and radiative forcing, flow of energy) must be given as a rate
```

**Comment by swillner** [comment]
Superseeded by #110

**Pull request closed without merging**